core/codegen/src/attribute/route/mod.rs

```diff
@@ -319,6 +352,9 @@ fn codegen_route(route: Route) -> Result<TokenStream> {
     let rank = Optional(route.attr.rank);
     let format = Optional(route.attr.format.as_ref());
 
+    // Get the doc comment
+    let docstring = &route.docstring;
```

Comment:
No need to separate this. Group it with the rest.

Reply:
Do you mean add it to the `StaticInfo::schemas` field?

Reply:
Sorry, I just need style wise. Basically to just remove the whitespace of and comment.

Reply:
Right okay